cmd/jag/commands/device.go

```diff
@@ -18,7 +18,7 @@ import (
 const (
 	JaguarDeviceIDHeader         = "X-Jaguar-Device-ID"
 	JaguarSDKVersionHeader       = "X-Jaguar-SDK-Version"
-	JaguarDisabledHeader         = "X-Jaguar-Disabled"
+	JaguarNetworkDisabledHeader  = "X-Jaguar-Network-Disabled"
```

Comment:
Maybe we should change this to something like `jag.wifi` and then have a positive flag that defaults to true, but can be turned off through `-D jag.wifi=false`?

Reply:
done.